lib/playground.dart

```diff
@@ -343,9 +343,14 @@ class Playground {
         return new Annotation(issue.kind, issue.message, issue.line,
             start: start, end: end);
       }).toList());
+
+      _updateRunButton(
+          hasErrors: result.issues.any((issue) => issue.kind == 'error'),
+          hasWarnings: result.issues.any((issue) => issue.kind == 'warning'));
```

Comment:
Yeah, the methods on List and Iterable - `where`, `any`, `map`, `join`, .. - are super useful.